stdlib/3/subprocess.pyi

```diff
@@ -671,9 +671,12 @@ else:
         errors: Optional[str] = ...,
     ) -> Any: ...  # morally: -> _TXT
 
-PIPE: int
-STDOUT: int
-DEVNULL: int
+_PIPE = Literal[-1]
```

Comment:
Curiously, it complains if I try to use `PIPE` as an annotation, claiming it is an invalid type, yet the overloads all work correctly despite the complaints. So, these aliases are here to suppress those errors.